README.md

```diff
@@ -1,3 +1,40 @@
 # gcmalloc
 
 A Rust allocator with support for conservative garbage collection.
+
+## The Drop Trait
+
+Destructors for `Gc`s will be ran from the "outside-in", this guarantee is given
```

Comment:
I'm not sure that the clause "this guarantee is given to us..." is correct (surely we have to do some work to make it happen?) or useful. It might be easier to say "As with normal `drop`, `Gc` destructors are run from the "outside-in"."?

Reply:
I've re-worded this and added an opening paragraph for context. I'm wary not to use the term _destructor_ interchangeably with _finalizer_ as GC terminology does make them distinct.

Reply:
I'm also aware that I should be spelling finalizer is with an _s_! :gb: But Java broke me from an early age and anything other than the American spelling looks weird to me.

Reply:
It's OK, "finaliser" is also acceptable British English, so you don't have to worry about betraying the Queen ;)